framework_tvb/tvb/adapters/analyzers/wavelet_adapter.py

```diff
@@ -66,12 +66,12 @@ class WaveletAdapterModel(ViewModel, ContinuousWaveletTransform):
 class RangeForm(Form):
     def __init__(self, prefix=''):
         super(RangeForm, self).__init__(prefix)
-        self.lo = SimpleFloatField(self, name='lo', required=True, label='Lo', doc='start of range')
-        # default=ContinuousWaveletTransform.frequencies.lo)
-        self.step = SimpleFloatField(self, name='step', required=True, label='Step', doc='step of range')
-        # default=ContinuousWaveletTransform.frequencies.step)
-        self.hi = SimpleFloatField(self, name='hi', required=True, label='Hi', doc='end of range')
-        # default=ContinuousWaveletTransform.frequencies.hi)
+        self.lo = SimpleFloatField(self, name='lo', required=True, label='Lo', doc='start of range',
```

Comment:
We should replace the usage of SimpleFloatField with FloatField and adapt the definition.